core/tests/puppeteer-acceptance-tests/utilities/user/super-admin.ts

```diff
@@ -27,6 +27,13 @@ const rolesSelectDropdown = 'div.mat-select-trigger';
 const addRoleButton = 'button.oppia-add-role-button';
 
 export class SuperAdmin extends BaseUser {
+  async navigateToAdminPage(): Promise<void> {
+    await this.goto(testConstants.URLs.AdminPage);
+  }
+
+  async navigateToRolesTab(): Promise<void> {
```

Comment:
Since the name of the URL constant is AdminPageRolesTab, I suggest renaming this function to navigateToAdminPageRolesTab

Reply:
updated.